net-irc/hexchat/hexchat-2.12.1.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+inherit fdo-mime gnome2-utils mono-env multilib python-single-r1
+
+DESCRIPTION="Graphical IRC client based on XChat"
+HOMEPAGE="https://hexchat.github.io/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="git://github.com/hexchat/hexchat.git"
+else
+	SRC_URI="https://dl.hexchat.net/hexchat/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+fi
+
+LICENSE="GPL-2 plugin-fishlim? ( MIT )"
+SLOT="0"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl lua nls perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
```

Comment:
@SoapGentoo I think you're reading it backwards. We've got plugins turned on by so that people don't need to enable it as much.

Regardless I think @nvinson is on the right track with simply removing the ability to disable plugin support. Enough of the functionality the use flags expose depends upon it that it may just be beneficial to remove the plugins USE flag.